### PR TITLE
Removed built-in Kubernetes manifest command #9421

### PR DESCRIPTION
**Pull request by aryanv175**
As stated in the issue #9421 created by Mr. Jonathon Yu, This will deprecate the original installation method and recommend that users use the Helm chart instead.

### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [ ] This pull request includes tests or only affects documentation.
- [x] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`, `docs`.

### CODE REVIEW

**Comment by madkinsz** [comment]
There's an existing pull request for this at https://github.com/PrefectHQ/prefect/pull/9434

**Comment by aryanv175** [comment]
Hello! #9434 only passed 6 checks. 

If you would like me to close this PR, let me know!

Looking forward to contributing more here!

**Comment by madkinsz** [comment]
https://github.com/PrefectHQ/prefect/pull/9434 looks stalled so we can move forward with this one — let's try to improve the message suggesting what the user should do. There is no 'helm server' command. Instead, we'll want to point them to our Helm chart repository.

**Comment by aryanv175** [comment]
So would you like me to change this help="Deprecated. Use 'helm server' instead.", to: help="Deprecated. Use 'helm chart repository' instead."

Let me know and I can make the change. If there is something you would want me to go through before making any more PRs let me know. My changes are failing the  unit tests. Is this the reason?

**Comment by madkinsz** [comment]
We used quotes previously to indicate another command to use. Here, we can just do something like "Deprecated. Use the Prefect Helm chart instead (https://github.com/PrefectHQ/prefect-helm)."

The `tests/cli/test_kubernetes_manifest.py::test_printing_the_job_base_manifest` test is failing because this warning is now included in stdout. We'll need to print deprecation warnings to stderr instead so this command doesn't break when piped to `kubectl`.

File: src/prefect/cli/kubernetes.py [hunk 1]
```diff
@@ -26,7 +26,11 @@
 
 manifest_app = PrefectTyper(
     name="manifest",
-    help="Commands for generating Kubernetes manifests.",
+    help="Deprecated. Use 'helm server' instead.",
+    deprecated=True,
+    deprecated_name="prefect kubernetes",
+    deprecated_start_date="May 2023",
+    deprecated_help="Use 'helm server' instead."
```
Comment (chrisguidry) [inline src/prefect/cli/kubernetes.py:33]:
```suggestion
    deprecated_help="Use 'helm server' instead.",
```

In order to adhere to `black` formatting, this requires a trailing comma.

Comment (madkinsz) [reply src/prefect/cli/kubernetes.py:33]: Additionally note that `'helm server'` is not an acceptable suggestion per https://github.com/PrefectHQ/prefect/pull/9479#issuecomment-1558307826

Comment (chrisguidry) [reply src/prefect/cli/kubernetes.py:33]: Ah yes, thanks, @madkinsz!

Comment (chrisguidry) [reply src/prefect/cli/kubernetes.py:33]: @aryanv175 Are you interested in completing the updates here?

**Pull request closed without merging**